.env_example

```diff
@@ -0,0 +1,2 @@
+GOLLUM_AUTHOR_USERNAME="Anonymous"
```

Comment:
Maybe add comments to this file to explain its usage? Is this just for use with `docker compose`, or does a 'normal' `docker run` also make use of the `.env` file?

Reply:
.env is only used with docker compose.